packages/daemon/package.json

```diff
@@ -60,20 +60,20 @@
     "debug": "^4.0.1",
     "eth-sig-util": "^2.1.0",
     "ethereumjs-tx": "^1.3.7",
-    "ethereumjs-util": "^6.0.0",
+    "ethereumjs-util": "^6.1.0",
     "ethereumjs-wallet": "^0.6.2",
     "fs-extra": "^7.0.0",
     "graphql": "^14.1.1",
     "graphql-relay": "^0.6.0",
     "multibase": "^0.6.0",
     "oboe": "^2.1.4",
     "rxjs": "^6.3.3",
-    "web3-provider-engine": "^14.0.6",
-    "web3-providers-http": "^1.0.0-beta.36",
+    "web3-providers-ws": "^1.0.0-beta.37",
     "web3-utils": "^1.0.0-beta.36"
   },
   "devDependencies": {
     "@oclif/dev-cli": "^1.21.2",
+    "babel-plugin-inline-json-import": "^0.3.2",
```

Comment:
I was getting an error during daemon build when trying to import json: `Cannot find module './inviteABI.json'` and adding `inline-json-import` plugin to babel.rc config was a solution I found, but maybe it's part of a different issue if that shouldn't be required.

Reply:
Does it work if you do `require("./inviteABI.json")` rather than `import`?
If so might be a Babel-specific change, no problem adding the plugin if it's the necessary workaround.

Reply:
Seems to be the same issue using `require`

Reply:
Is the file present at the right path? Maybe I missed the news if node dropped support for importing JSON files directly, anyways if it's work with the babel plugin no need to overthink it.